sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -495,13 +495,16 @@ case class TruncateTableCommand(
 }
 
 /**
- * Command that looks like
+ * Commands can of the following forms.
  * {{{
  *   DESCRIBE [EXTENDED|FORMATTED] table_name partitionSpec?;
+ *   DESCRIBE QUERY <query>
+ *   DESCRIBE <query>
  * }}}
  */
 case class DescribeTableCommand(
```

Comment:
As this is named as `DescribeTableCommand`, shall we have individual command for describing query?

Reply:
Otherwise, shall we rename it to `DescribeTableOrQueryCommand`?

Reply:
@viirya We could. Just that there are not that much changes. Lets see if others feel the same way. If so, i would change it.